### PR TITLE
Fixing footnotes and block quotes 

### PR DESCRIPTION
**Pull request by catlook**
Also adding more use cases and fixing extraneous markdown mistakes.  also experimented with the description metadata format.

### CODE REVIEW

File: content/cloud-overview/cloud-economics.md [hunk 1]
```diff
@@ -9,8 +9,9 @@ last_modified_date: '2017-02-28'
 last_modified_by: Cat Lookabaugh
 product: Cloud Overview
 product_url: cloud-overview
-description: >
-  This is a description of Economics. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi efficitur massa vel vestibulum fringilla. Fusce aliquet risus urna, sit amet efficitur eros convallis suscipit. Vivamus aliquam massa posuere commodo cursus. Duis ut pretium 40.
+description: > Is cost reduction the only reason to adopt a cloud hosting
```
Comment (nimbinatus) [inline content/cloud-overview/cloud-economics.md]: Here, we need either no `>` or the description needs to start on the next line. This is preventing the Strider build from generating a preview:
> Error reading file /workspace/data/rackerlabs-rackspace-whitepapers-master/job-58d5a9c72c42040100e2319a/content/cloud-overview/cloud-economics.md: (<unknown>): did not find expected comment or line break while scanning a block scalar at line 12 column 14

File: content/cloud-overview/101-guide-to-the-cloud.md [hunk 2]
```diff
@@ -9,8 +9,8 @@ last_modified_date: '2017-02-22'
 last_modified_by: Cat Lookabaugh
 product: Cloud Overview
 product_url: cloud-overview
-description: >
-  This is a description of 101. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras mollis tincidunt mi, at fringilla nulla vulputate quis. Sed eleifend purus eu mattis placerat. 30.
+description: > Could you describe what "the cloud" actually is if someone asked?  Do you
```
Comment (nimbinatus) [inline content/cloud-overview/101-guide-to-the-cloud.md]: Here, we need either no `>` or the description needs to start on the next line. This is preventing the Strider build from generating a preview:
> Error reading file /workspace/data/rackerlabs-rackspace-whitepapers-master/job-58d5a9c72c42040100e2319a/content/cloud-overview/101-guide-to-the-cloud.md: (<unknown>): did not find expected comment or line break while scanning a block scalar at line 12 column 14

File: content/cloud-overview/the-cloud-at-your-pace.md [hunk 3]
```diff
@@ -9,8 +9,10 @@ last_modified_date: '2017-03-06'
 last_modified_by: Cat Lookabaugh
 product: Cloud Overview
 product_url: cloud-overview
-description: >
-  This is a description of Pace. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis rutrum nulla. Nunc a sodales leo, et blandit lectus. Quisque lorem justo, imperdiet sit amet libero eu, sodales suscipit eros. Suspendisse aliquet magna lorem, sed accumsan ante venenatis 45.
+description: > The cloud has taken many twists and turns on its route to maturity. Business
```
Comment (nimbinatus) [inline content/cloud-overview/the-cloud-at-your-pace.md]: Here, we need either no `>` or the description needs to start on the next line. This is preventing the Strider build from generating a preview:
> Error reading file /workspace/data/rackerlabs-rackspace-whitepapers-master/job-58d5a9c72c42040100e2319a/content/cloud-overview/the-cloud-at-your-pace.md: (<unknown>): did not find expected comment or line break while scanning a block scalar at line 12 column 14

File: content/cloud-overview/cloudonomics-the-economics-of-cloud-computing.md [hunk 4]
```diff
@@ -58,7 +58,7 @@ themselves. The diagram below illustrates the extent that routine and
 non-core tasks, like patching operating systems and performing backups,
 impact upon the time of IT departments.
 
-![](http://c0179631.cdn.cloudfiles.rackspacecloud.com/cloudonomics.png)
+<img src="{% asset_path cloud-overview/cloudonomics-the-economics-of-cloud-computing/cloudonomics.png %}" />
```
Comment (nimbinatus) [inline content/cloud-overview/cloudonomics-the-economics-of-cloud-computing.md]: This image appears to be missing (or the path may need correcting):
> Couldn't find file 'cloud-overview/cloudonomics-the-economics-of-cloud-computing/cloudonomics.png' in content/cloud-overview/cloudonomics-the-economics-of-cloud-computing.md

File: content/cloud-overview/hybrid-hosting-evolving-the-cloud-in-2011.md [hunk 5]
```diff
@@ -327,8 +325,7 @@ launches and live events, both of which dramatically increase traffic
 for a short period of time. Just as important, MLG minimized the costs
 of unused capacity between peaks.
 
-**[Click here to see full case
-study.](/how-to/rackspace-gives-major-league-gaming-mlg-the-power-of-scale)**
+**[Click here to see full case study.](/how-to/rackspace-gives-major-league-gaming-mlg-the-power-of-scale)**
```
Comment (catlook) [inline content/cloud-overview/hybrid-hosting-evolving-the-cloud-in-2011.md]: excellent question...lemme see if that one even exists in the new world order.  Some whitepapers are not being converted

File: content/professional-services/how-the-cloud-accelerates-innovation.md [hunk 6]
```diff
@@ -354,9 +354,10 @@ more.*
 
 <sup>1</sup> V. Yeo, “Dell touts hefty savings from IT efficiencies.” ZDNet Asia,
 March 3, 2010: [http://www.zdnetasia.com/dell-touts-
-hefty-savings-from-it-efficiencies-62061584.htm](http://www.zdnetasia.com/dell-touts-%20hefty-savings-from-it-efficiencies-62061584.htm)\
+hefty-savings-from-it-efficiencies-62061584.htm](http://www.zdnetasia.com/dell-touts-%20hefty-savings-from-it-efficiencies-62061584.htm)
+
 <sup2</sup> AgencyNet Case Study, Rackspace,
-2011: [/how-to/rackspace-keeps-nickelodeons-kids-choice-awards-virtual-slime-flowing](/how-to/rackspace-keeps-nickelodeons-kids-choice-awards-virtual-slime-flowing)\
+2011: [/how-to/rackspace-keeps-nickelodeons-kids-choice-awards-virtual-slime-flowing](/how-to/rackspace-keeps-nickelodeons-kids-choice-awards-virtual-slime-flowing)
```
Comment (catlook) [inline content/professional-services/how-the-cloud-accelerates-innovation.md]: ditto

File: content/use-cases/ebs-says-goodbye-to-email-headaches-with-rackspace.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
+---
+permalink: ebs-says-goodbye-to-email-headaches-with-rackspace/
+audit_date:
+title: EBS Says Goodbye to Email Headaches with Rackspace
```
Comment (catlook) [inline content/use-cases/ebs-says-goodbye-to-email-headaches-with-rackspace.md]: LOL.  don't see why not!  Let me make the appropriate changes to meta and directories

File: content/use-cases/firefly-digital-glows-for-rackspace.md [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+worrying' to 'let's talk about solutions.' I threw every concern I had
+at these guys, and they had answers. They were great."
+
+What was immediately clear to Dalton was that the professionals at
+Rackspace knew exactly how to deal with the kinds of issues that were
+waking him up at night. Rackspace could provide the hardware, the
+services and the Fanatical Support that Firefly Digital needed to keep
+its customers up and running around the clock.
+
+The proposal they've put together addresses every concern I raised;
```
Comment (catlook) [inline content/use-cases/firefly-digital-glows-for-rackspace.md]: I think there weren't originally, but let me compare to production.

File: content/use-cases/fueling-the-rapid-rise-of-kendra-scott-design-requires-high-performance-ecommerce.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
+---
+permalink: fueling-the-rapid-rise-of-kendra-scott-design-requires-high-performance-ecommerce/
+audit_date:
+title: "Fueling the Rapid Rise of Kendra Scott Design\L Requires High-Performance Ecommerce"
```
Comment (catlook) [inline content/use-cases/fueling-the-rapid-rise-of-kendra-scott-design-requires-high-performance-ecommerce.md]: fixing.

File: content/use-cases/fueling-the-rapid-rise-of-kendra-scott-design-requires-high-performance-ecommerce.md [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+type: case_study
```
Comment (catlook) [inline content/use-cases/fueling-the-rapid-rise-of-kendra-scott-design-requires-high-performance-ecommerce.md]: Thought i'd already changed the type to whitepaper.  If not, I'll correct it.

File: content/use-cases/fundapps-delivers-compliance-software-with-rackspace-hybrid-cloud.md [hunk 11]
```diff
@@ -0,0 +1,131 @@
+---
+permalink: fundapps-delivers-compliance-software-with-rackspace-hybrid-cloud/
+audit_date:
+title: FundApps Delivers Compliance Software with Rackspace Hybrid Cloud
+type: case_study
```
Comment (catlook) [inline content/use-cases/fundapps-delivers-compliance-software-with-rackspace-hybrid-cloud.md]: will do.

File: content/use-cases/future-buro-runs-big-campaigns-in-the-cloud.md [hunk 12]
```diff
@@ -0,0 +1,142 @@
+---
+permalink: future-buro-runs-big-campaigns-in-the-cloud/
+audit_date:
+title: Future B&uuml;ro Runs Big Campaigns in the Cloud
+type: case_study
```
Comment (catlook) [inline content/use-cases/future-buro-runs-big-campaigns-in-the-cloud.md]: will do

File: content/use-cases/gallant-fx-finds-on-demand-support-for-an-on-demand-market-via-rackspace.md [hunk 13]
```diff
@@ -0,0 +1,140 @@
+---
+permalink: gallant-fx-finds-on-demand-support-for-an-on-demand-market-via-rackspace/
+audit_date:
+title: Gallant FX Finds On-demand Support for an On-demand Market via Rackspace
+type: case_study
```
Comment (catlook) [inline content/use-cases/gallant-fx-finds-on-demand-support-for-an-on-demand-market-via-rackspace.md]: will do

File: content/use-cases/globalscape-uses-rackspace-hosting-to-deliver-award-winning-products-to-customers.md [hunk 14]
```diff
@@ -0,0 +1,103 @@
+---
+permalink: globalscape-uses-rackspace-hosting-to-deliver-award-winning-products-to-customers/
+audit_date:
+title: Globalscape Uses Rackspace&reg; Hosting To Deliver Award-Winning Products To Customers
+type: case_study
```
Comment (catlook) [inline content/use-cases/globalscape-uses-rackspace-hosting-to-deliver-award-winning-products-to-customers.md]: will do

File: content/use-cases/intervalzero-has-zero-email-headaches-with-rackspace.md [hunk 15]
```diff
@@ -0,0 +1,91 @@
+---
+permalink: a-high-street-alternative-in-the-rackspace-hybrid-cloud/
```
Comment (catlook) [inline content/use-cases/intervalzero-has-zero-email-headaches-with-rackspace.md]: Opps...thought i caught all those.  fixing

File: content/use-cases/ilium-software-trades-email-server-for-fanatical-support.md [hunk 16]
```diff
@@ -0,0 +1,131 @@
+---
+permalink: a-high-street-alternative-in-the-rackspace-hybrid-cloud/
```
Comment (catlook) [inline content/use-cases/ilium-software-trades-email-server-for-fanatical-support.md]: Opps...thought i caught all those.  fixing

File: content/use-cases/hr-softwares-next-generation-finds-footing-in-the-cloud.md [hunk 17]
```diff
@@ -0,0 +1,125 @@
+---
+permalink: hr-softwares-next-generation-finds-footing-in-the-cloud/
+audit_date:
+title: "HR Software's Next Generation Finds Footing In The Cloud
```
Comment (catlook) [inline content/use-cases/hr-softwares-next-generation-finds-footing-in-the-cloud.md]: doh.  fixing

File: content/cloud-overview/cloud-economics.md (hunk 1 above)
Comment (catlook) [inline content/cloud-overview/cloud-economics.md]: new line it is!

File: content/cloud-overview/101-guide-to-the-cloud.md (hunk 2 above)
Comment (catlook) [inline content/cloud-overview/101-guide-to-the-cloud.md]: new line it is!

File: content/cloud-overview/the-cloud-at-your-pace.md (hunk 3 above)
Comment (catlook) [inline content/cloud-overview/the-cloud-at-your-pace.md]: new line it is!

File: content/cloud-overview/cloudonomics-the-economics-of-cloud-computing.md (hunk 4 above)
Comment (catlook) [inline content/cloud-overview/cloudonomics-the-economics-of-cloud-computing.md]: will add!

File: content/cloud-overview/hybrid-hosting-evolving-the-cloud-in-2011.md (hunk 5 above)
Comment (catlook) [inline content/cloud-overview/hybrid-hosting-evolving-the-cloud-in-2011.md]: done

File: content/professional-services/how-the-cloud-accelerates-innovation.md (hunk 6 above)
Comment (catlook) [inline content/professional-services/how-the-cloud-accelerates-innovation.md]: done

File: content/use-cases/ebs-says-goodbye-to-email-headaches-with-rackspace.md (hunk 7 above)
Comment (catlook) [inline content/use-cases/ebs-says-goodbye-to-email-headaches-with-rackspace.md]: done

File: content/use-cases/firefly-digital-glows-for-rackspace.md (hunk 8 above)
Comment (catlook) [inline content/use-cases/firefly-digital-glows-for-rackspace.md]: Sure enough, no quotes in original...but definitely should be.  Done.

File: content/use-cases/fueling-the-rapid-rise-of-kendra-scott-design-requires-high-performance-ecommerce.md (hunk 10 above)
Comment (catlook) [inline content/use-cases/fueling-the-rapid-rise-of-kendra-scott-design-requires-high-performance-ecommerce.md]: done

File: content/use-cases/fundapps-delivers-compliance-software-with-rackspace-hybrid-cloud.md (hunk 11 above)
Comment (catlook) [inline content/use-cases/fundapps-delivers-compliance-software-with-rackspace-hybrid-cloud.md]: done

File: content/use-cases/future-buro-runs-big-campaigns-in-the-cloud.md (hunk 12 above)
Comment (catlook) [inline content/use-cases/future-buro-runs-big-campaigns-in-the-cloud.md]: done

File: content/use-cases/gallant-fx-finds-on-demand-support-for-an-on-demand-market-via-rackspace.md (hunk 13 above)
Comment (catlook) [inline content/use-cases/gallant-fx-finds-on-demand-support-for-an-on-demand-market-via-rackspace.md]: done

File: content/use-cases/globalscape-uses-rackspace-hosting-to-deliver-award-winning-products-to-customers.md (hunk 14 above)
Comment (catlook) [inline content/use-cases/globalscape-uses-rackspace-hosting-to-deliver-award-winning-products-to-customers.md]: done

File: content/use-cases/intervalzero-has-zero-email-headaches-with-rackspace.md (hunk 15 above)
Comment (catlook) [inline content/use-cases/intervalzero-has-zero-email-headaches-with-rackspace.md]: done

File: content/use-cases/ilium-software-trades-email-server-for-fanatical-support.md (hunk 16 above)
Comment (catlook) [inline content/use-cases/ilium-software-trades-email-server-for-fanatical-support.md]: done

File: content/use-cases/hr-softwares-next-generation-finds-footing-in-the-cloud.md (hunk 17 above)
Comment (catlook) [inline content/use-cases/hr-softwares-next-generation-finds-footing-in-the-cloud.md]: done

File: content/cloud-overview/cloud-economics.md (hunk 1 above)
Comment (catlook) [inline content/cloud-overview/cloud-economics.md]: done

File: content/cloud-overview/101-guide-to-the-cloud.md (hunk 2 above)
Comment (catlook) [inline content/cloud-overview/101-guide-to-the-cloud.md]: done

File: content/cloud-overview/the-cloud-at-your-pace.md (hunk 3 above)
Comment (catlook) [inline content/cloud-overview/the-cloud-at-your-pace.md]: done

File: content/cloud-overview/cloudonomics-the-economics-of-cloud-computing.md (hunk 4 above)
Comment (catlook) [inline content/cloud-overview/cloudonomics-the-economics-of-cloud-computing.md]: Dang it.  I'll pulled the graphics onto my desktop...just hadn't dropped them in folder.  I was in a rush....that'll teach me!  Done.

**Comment by catlook** [comment]
Ok fixes are in...I don't see the preview link in here.  Should I?

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-da61c8b238/whitepapers/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-4a8a147cef/whitepapers/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b6570b0a1f/whitepapers/)

**Comment by nimbinatus** [comment]
LGTM, +1

**Pull request merged**